### PR TITLE
PB-315 Document Bitbucket Server setup.

### PR DESCRIPTION
**Pull request by jkaeser**
https://jira.zivtech.com/browse/PB-315

### CODE REVIEW

**Comment by jkaeser** [comment]
To review:

1. Visit `/git/bitbucket-server` and make sure all the information is accurate and understandable.
2. Ensure the lightbox-enabled images work well.
3. (Optional) Quickly glance through the site to find outdated references to Bitbucket Server or Stash. I did a search and the only thing I didn't adjust was the `about.md` file, which is not linked to anywhere in the docs. I'll address this file in a different Jira ticket.

File: docs/git/bitbucket-server.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  <img src="/images/bitbucket-server/bbserver-setup-2--configure-app-link-url.png" alt="" class="full-width">
+  </a>
+
+3. Give your application a logical name, such as **Probo.CI**, and leave the rest of the application settings blank.
+  <a href="/images/bitbucket-server/bbserver-setup-3--configure-app-link-name.png" data-lightbox="bbserver-setup-3">
+  <img src="/images/bitbucket-server/bbserver-setup-3--configure-app-link-name.png" alt="" class="full-width">
+  </a>
+
+<a name="configure-auth"></a>
+## Configure authentication settings.
```
Comment (tortillaj) [inline docs/git/bitbucket-server.md:38]: The headings automatically have ids for you. You don't have to put in line 39, and you can change line 13 to `<a href="#configure-authentication-settings"> .... </a>`. Similar story for the other 3.

**Comment by tortillaj** [comment]
Will you pull out line 15 here: https://github.com/ProboCI/probo-docs/blob/b5057a9facbe04962817e6c2b4a7073ff6be3eff/_includes/head.html#L15

The style guide is now coming in line 13 just above it, meaning it'll come in twice unless we remove 15

File: _stylesheets/01-utilities/_utility.scss [hunk 2]
```diff
@@ -27,7 +27,8 @@ img {
 	max-width: 100%;
 
 	&.full-width {
-		width: auto;
+    padding: $base-spacing 0;
```
Comment (lliss) [inline _stylesheets/01-utilities/_utility.scss:16]: Spacing is weird.

**Comment by jkaeser** [comment]
It seems [the current GH outage](https://twitter.com/githubstatus/status/806228936208289792) is preventing my rebase from making the trip upstream.

**Comment by jkaeser** [comment]
There we go! This is ready to merge once again. 😎

File: _stylesheets/01-utilities/_utility.scss (hunk 2 above)
Comment (tortillaj) [reply _stylesheets/01-utilities/_utility.scss:16]: @jkaeser agree with above ^ ... can we just get a reformat of this whole file? Also, can probably remove [lines 1-13](https://github.com/ProboCI/probo-docs/blob/5f0b525ba82038d04b702b48902a6675a2973fa4/_stylesheets/01-utilities/_utility.scss#L1-L13)

**Pull request merged**